ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/static_routes.schema.yml

```diff
@@ -19,6 +19,12 @@ keys:
         gateway:
           type: str
           description: IPv4 Address
+        track:
```

Comment:
Instead of a string with a single valid value, wouldn't a boolean make more sense here? Like `track_bfd: <bool>`?